test/snapshots/v0-80212-homeowner-80000-joint-4.json

```diff
@@ -3,7 +3,7 @@
   "is_under_150_ami": true,
   "is_over_150_ami": false,
   "pos_savings": 14000,
-  "tax_savings": 5836,
+  "tax_savings": 5523,
```

Comment:
This is actually a relic of #665. It turns out the v0 snapshot test wasn't actually testing the equality of the _entire_ response, so this snapshot test didn't fail on #665, so I didn't update the snapshots on that PR.